### ISSUE TITLE
Issues when installing on Ubuntu 14.04 (Ruby 1.9.3)

### ISSUE THREAD

**Issue by slaponicus** [sportdb/sport.db.tools#1]
I successfully installed sportdb on Ubuntu 14.04 LTS running Ruby 1.9.3 but encountered a few issues:

-The activerecord 4.2.7 and datafile dependencies must be manually installed. Rubygems tries to install activerecord 5+ by default which doesn't work with sportdb and I get an activesupport error until I install datafile (this error occurs when it's building the docs - does this something to do with textutils?).

-When installation completes, I still get the error "Error installing sportdb: activesupport requires Ruby version >= 2.2.2" despite the fact that sportdb installs and runs fine. 

Again, after all of this sportdb is usuable and works great - I just wanted to save someone some trouble in case they're in a similar situation.

**Comment by geraldb** [comment]
@slaponicus  Thanks for testing, trying and reporting.  Not (yet) working with activerecord 5+ is a known issue. The plan (of course) is to update the sportdb machinery for activerecord 5+ (it's an issue with the new "auto-index" creation in activerecord 5 leading to duplicate indices).   Cheers.

**Comment by randomm** [comment]
Perhaps this is a related issue?

    *** error: Index name 'index_taggings_on_tag_id' on table 'taggings' already exists

I'm getting that with a fresh install (uses activerecord 5.0.0.1 out of the box) when trying to create a new database.

**Comment by slaponicus** [comment]
@randomm yeah, right now (in my experience) you have to gem install activerecord 4.2.7 and datafile manually before installing sportdb

**Comment by randomm** [comment]
Yup that did the trick. Installing either sportdb or datafiles gem introduced active* 5.0.0.1 gems into my system, but deleting them manually worked. Thanks!
